apps/ui/src/composables/useWeb3.ts

```diff
@@ -95,9 +94,10 @@ export function useWeb3() {
       const acc = accounts.length > 0 ? accounts[0] : null;
 
       if (acc) {
-        const names = await getNames([acc]);
+        const usersStore = useUsersStore();
+        await usersStore.fetchUser(acc);
         state.account = formatAddress(acc);
-        state.name = names[acc];
+        state.name = usersStore.getUser(acc)?.name || '';
```

Comment:
To use the user's `username` set in the profile when available. This will show the username on the page top right user menu

![Screenshot 2024-06-17 at 19 33 11](https://github.com/snapshot-labs/sx-monorepo/assets/495709/ccfe0ec6-8502-473e-bed2-714f8f70f795)